ucm2/Intel/sof-hda-dsp/HiFi.conf

```diff
@@ -8,7 +8,7 @@ SectionVerb {
 	Value.TQ "HiFi"
 }
 
-Include.hda-analog.File "/HDA/HiFi-analog.conf"
+Include.hda-analog.File "/Intel/sof-hda-dsp/HiFi-analog-sof.conf"
```

Comment:
I can't recall why the initial part was 'HDA' but it was intentional.

I don't fully understand the impact of renaming the path, but this is potentially problematic.

Reply:
Thanks @perexg , that would be better way indeed, and there's lot of repeats. I'll do that for next update.

Reply:
> Also, if something repeats, you may consider to use macros...

@perexg I don't seem to be able to use macros inside `EnableSequence []`. I get e.g. error `ALSA lib parser.c:1237:(parse_sequence) error: sequence command 'Macro.id2.HeadphoneSofDrcEqEnable' is ignored` if I try such.